spec/models/event_spec.rb

```diff
@@ -346,4 +346,28 @@
       expect(other_event.week).to eq 48
     end
   end
+
+  describe 'current?' do
+    let(:past_event) { create(:event, program: conference.program) }
+    let(:future_event) { create(:event, program: conference.program) }
+
+    before do
+      event.update_attributes(start_time: Time.current.to_s)
+      past_event.update_attributes(start_time: (Time.current - 2.days).to_s)
```

Comment:
Why not already in the factory?

``
let(:past_event) { create(:event, program: conference.program, start_time: (Time.current - 2.days)) }
``
https://github.com/brennovich/cheat-ruby-sheets/blob/master/factory_girl.md#building-factories